recrypt/recrypt.c

```diff
@@ -15,6 +15,7 @@
 #include "tarsnap_opt.h"
 #include "tsnetwork.h"
 #include "warnp.h"
+#include "imalloc.h"
```

Comment:
Includes should be sorted in alphabetical order within each block -- this should be between `#include "crypto.h"` and `#include "keyfile.h"`.